src/netex_idf/stops.rs

```diff
@@ -83,8 +107,101 @@ fn load_stop_areas(elem: &Element) -> Result<CollectionWithId<StopArea>> {
             );
         }
     }
+}
 
-    Ok(stop_areas)
+fn load_coords(quay: &Element) -> Result<(f64, f64)> {
+    let gml_pos = quay
+        .try_only_child("Centroid")?
+        .try_only_child("Location")?
+        .try_only_child("pos")?
+        .text()
+        .trim()
+        .to_string();
+
+    let coords: Vec<&str> = gml_pos.split_whitespace().collect();
+    if coords.len() != 2 {
+        bail!("longitude and latitude not found");
+    }
+
+    Ok((coords[0].parse()?, coords[1].parse()?))
+}
+
+fn load_stop_points<'a>(
+    members: &[&'a Element],
```

Comment:
This seems to be a list of `Quay` and `StopPlace`. Why not directly send the list of `Quay` here (and send the list of `StopPlace` to the function `load_stop_areas` above)?

Reply:
done